controllers/drplacementcontrol_controller.go

```diff
@@ -1084,7 +1081,7 @@ func (d *DRPCInstance) switchToFailoverCluster() (bool, error) {
 
 	d.advanceToNextDRState()
 	d.log.Info("Exiting runFailover", "state", d.getLastDRState())
-	setMetricsTimerFromDRState(rmn.FailingOver, timerStop)
+	d.setMetricsTimerFromDRState(rmn.FailingOver, timerStop)
```

Comment:
THis should change to FailedOver, FailedBack, Relocated, Deployed respectively to eliminate confusion.  I reworked the function `setMetricsTimerFromDRState` above to show what I mean.

Reply:
I thought about this a bit, and agree with the idea that this is clearer. When calling `setMetricsTimerFromDRState`, now only the DR state is used. Internally, the timer starting and stopping is handled by the switch statement above. Hopefully this is easier to use now.